### ISSUE TITLE
NTR: Malic acid high in urine (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8614]
New term request Malic acid high in urine (segal_030513182745): Malate  UMLS: 3276835

**Comment by pnrobinson** [comment]
Child of Dicarboxylic aciduria HP:0003215

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6001000
- label: Elevated urine malic acid level
- parents: [Dicarboxylic aciduria (HP:0003215)](https://hpo.jax.org/app/browse/term/HP:0003215)
- pmid: 
- def: The amount of malic acid in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8614
- synonyms:
